total_scattering/autogrouping/similarity.py

```diff
@@ -0,0 +1,90 @@
+# code it
+# Bundle all of this into a class, rename it something better
+import numpy
+
+'''
+This file calculates similarity metrics for numpy vectors.
+It is build from cross correlations
+'''
+
+
+class similarity_metric:
+
+    def __init__(self, L=5):
+
+        self.L = L
+        self.weights = [self.weight_triangular(r) for r in range(0, self.L + 1)]
+
+    '''
+    symmetric weight function
+    '''
```

Comment:
Do you mind to move the above comment so that it is a docstring under the `weight_trriangular` function?